suite-native/module-send/src/screens/SendOutputsScreen.tsx

```diff
@@ -117,32 +118,27 @@ export const SendOutputsScreen = ({
     } = form;
     const watchedFormValues = useWatch({ control });
 
-    const storeFormDraftIfValid = useCallback(() => {
-        dispatch(
-            sendFormActions.storeDraft({
-                accountKey,
-                formState: constructFormDraft(getValues()),
-            }),
-        );
-    }, [accountKey, dispatch, getValues]);
+    const precomposeTransactionFeeLevels = useCallback(async () => {
+        const formState = constructFormDraft(getValues());
 
-    useEffect(() => {
-        if (watchedFormValues && isValid) debounce(storeFormDraftIfValid);
-    }, [isValid, storeFormDraftIfValid, watchedFormValues, debounce]);
+        if (!account) return;
```

Comment:
Would it make sense to move this at the very beginning of the callback block and change like this?
```suggestion
        if (!account || !networkFeeInfo) return;
```